packages/ttsc-lint-plugin-wp/tests/fixtures/parity/fixture.tsx

```
@@ -244,3 +244,113 @@ void closureReferenceCountsAsUsage;
+const objectRenderMethod = {
+  render() {
+    return <>{window}</>;
+  },
+};
+const namedWindowExpression = function window() {
+  return <>{window}</>;
+};
+function switchScopedDomGlobal(value) {
+  switch (value) {
+    case 1:
+      const window = 1;
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)